README.md

```diff
@@ -783,14 +946,25 @@ In such cases, you can generate sample user accounts using the steps below. This
     chmod +x insert_sample_data.sh && \
     ./insert_sample_data.sh
     ```
+-   **Windows**
+
+    ```
+    curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/feature/sample_data_scripts/documentation/1.0.0/scripts/windows/insert_sample_data.bat && insert_sample_data.bat
+    ```
 
     After successfully running the script mentioned above, the following user accounts will be created and available for login:
    
-| Email ID                 | Password   | Role                      |
-| ------------------------ | ---------- | ------------------------- |
-| aaravpatel@example.com   | Password1@ | state_educational_officer |
-| arunimareddy@example.com | Password1@ | state_educational_officer |
-| devikasingh@example.com  | Password1@ | state_educational_officer |
+      | Email ID                 | Password   | Role                      |
```

Comment:
Check the alignment of this table. It should be more towards left of the documentation.